packages/ui/useScroll/useScroll.js

```diff
@@ -1018,42 +1116,64 @@ const useScrollBase = (props) => {
 		}
 	}
 
-	function getReachedEdgeInfo () {
-		const
-			bounds = getScrollBounds(),
-			reachedEdgeInfo = {bottom: false, left: false, right: false, top: false};
+	// scroll start/stop
```

Comment:
This comment is to distinguish code sections due to a large number of lines, and it looks that this line does not work its role now. If it is hard to gather functions for 'start' and 'stop', what about regrouping code and add some new comments for sections?

Reply:
Yes, I know it got messy and I tried not to affect the semantics logic. The thing is that wrapping functions in useCallback forced me to move them in order to be defined before being used. However, I will try to group them and add new comments.